src/sider/aktivitetskrav/vurdering/InnstillingOmStansSkjema.tsx

```diff
@@ -0,0 +1,125 @@
+import { useAktivitetskravNotificationAlert } from "@/sider/aktivitetskrav/useAktivitetskravNotificationAlert";
+import { FormProvider, useForm } from "react-hook-form";
+import { StansdatoDatePicker } from "@/sider/aktivitetskrav/vurdering/StansdatoDatePicker";
+import { useAktivitetskravVurderingDocument } from "@/hooks/aktivitetskrav/useAktivitetskravVurderingDocument";
+
+const texts = {
+  sendInnstilling: "Send innstilling",
+  avbryt: "Avbryt",
+  title: "Innstilling om stans",
+  form: {
+    begrunnelseLabel: "Begrunnelse (obligatorisk)",
+    begrunnelseDecsription:
+      "Skriv kort om hvilke opplysninger som ligger til grunn for stans av sykepenger, samt din vurdering av hvorfor aktivitetsplikten ikke er oppfylt og vurdering av eventuelle nye opplysninger.",
+    missingBegrunnelse: "Vennligst angi begrunnelse",
+  },
+  afterSendInfo: {
+    title: "Etter innstilling er sendt må du:",
+    gosysoppgave: "Sende oppgave til NAY i Gosys.",
+    stoppknapp:
+      "Gi beskjed om stans til ny saksbehandlingsløsning via Stoppknappen under fanen Sykmeldinger i Modia.",
+    innstillingenBlirJournalfort:
+      "Innstillingen blir journalført og kan sees i Gosys.",
+  },
+  subtitle1:
+    "Innstilling må skrives og sendes til NAY i Gosys. Ved å lagre fjerner du hendelsen fra oversikten.",
+};
+
+const begrunnelseMaxLength = 5000;
+
+export interface FormValues {
+  stansFom: Date;
+  begrunnelse: string;
+}
+
+interface Props {
+  aktivitetskravUuid: string;
+  varselSvarfrist: Date;
+}
+
+export default function InnstillingOmStansSkjema({
+  aktivitetskravUuid,
+  varselSvarfrist,
+}: Props) {
+  const formMethods = useForm<FormValues>();
+  const {
+    register,
+    watch,
+    formState: { errors },
+    handleSubmit,
+  } = formMethods;
+  const vurderAktivitetskrav = useVurderAktivitetskrav(aktivitetskravUuid);
+  const { innstillingOmStansDocument } = useAktivitetskravVurderingDocument();
+  const { displayNotification } = useAktivitetskravNotificationAlert();
+
+  function submit(values: FormValues) {
+    const createAktivitetskravVurderingDTO: InnstillingOmStansVurderingDTO = {
+      status: AktivitetskravStatus.INNSTILLING_OM_STANS,
+      stansFom: values.stansFom,
+      beskrivelse: values.begrunnelse,
+      document: innstillingOmStansDocument({
+        stansDato: values.stansFom,
+        begrunnelse: values.begrunnelse,
+      }),
+    };
+    vurderAktivitetskrav.mutate(createAktivitetskravVurderingDTO, {
+      onSuccess: () => {
+        displayNotification(AktivitetskravStatus.INNSTILLING_OM_STANS);
+      },
+    });
+  }
+
+  return (
+    <FormProvider {...formMethods}>
+      <form onSubmit={handleSubmit(submit)} className="gap-4 flex flex-col">
+        <Heading level="2" size="small" className="mt-4">
+          {texts.title}
+        </Heading>
+        <BodyShort size="small">{texts.subtitle1}</BodyShort>
+        {vurderAktivitetskrav.isError && (
```

Comment:
Denne bør heller ligge f.eks. over submit-knappen i stedet for mellom skjemafeltene.

Reply:
Enig! Bra catch 🎣